doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -986,26 +981,20 @@ Persistence of notations
 
 Notations disappear when a section is closed.
 
-.. cmd:: Local Notation @notation
-
-   Notations survive modules unless the command ``Local Notation`` is used instead
-   of :cmd:`Notation`.
-
-.. cmd:: Local Declare Custom Entry @ident
-
-   Custom entries survive modules unless the command ``Local Declare
-   Custom Entry`` is used instead of :cmd:`Declare Custom Entry`.
-
 .. _Scopes:
 
 Interpretation scopes
 ----------------------
 
+   .. todo Would "notation scope" be a better term?  Do we have multiple types of scope?
```

Comment:
"notation scopes" suggests notations; "interpretation scopes" is much more vague.  Let's see what @herbelin thinks.

Reply:
I used to avoid scopes which is too general and I called them interpretation scopes (implicitly scopes for interpretation of notations). Notation scopes would be good and more focussed than interpretation scopes. As you prefer.